lib/gollum/templates/search.mustache

```diff
@@ -41,7 +38,7 @@
 
 {{#no_results}}
   <p id="no-results">
-    {{t.no_results}} <strong>{{query}}</strong>.
+    {{t.no_results}}
```

Comment:
Removing the query from here has caused one test to fail.

What is the reason for removing `{{query}}` here? It can be helpful for the user to see what their previous search query was—in case they made a mistake.

Reply:
The query is already in 'search_results_for', and it is unnecessary in 'no_results'.

![image](https://github.com/gollum/gollum/assets/23275497/74e5f4a2-e6d5-4ce4-9a04-4d39a527ba9b)

Reply:
I see now. Thanks for explaining!